kubernetes/cray-kiali/values.yaml

```diff
@@ -34,15 +34,15 @@ global:
 kiali-operator:
   image:
     repo: artifactory.algol60.net/csm-docker/stable/quay.io/kiali/kiali-operator
-    tag: v1.41.0
+    tag: v1.75.0
     pullPolicy: IfNotPresent
   resources:
     requests:
       cpu: "1000m"
       memory: "200Mi"
     limits:
       cpu: "2000m"
-      memory: "1Gi"
+      memory: "4Gi"
```

Comment:
The resource limit has been updated to `4Gi`
Since it was getting OOM Killed @ `1Gi`
At` 3Gi` also it was able to run but it had few restarts so I chose 4Gi as the final value.
Any feedback is appreciated!

Reply:
That is fine, I am curious if you know if this is an occasional spike in memory usage, or a constant state?

Reply:
Only when the operator starts in the beginning before deploying the kiali crd 
After that usage is normal. There is no memory leak, I have confirmed this, through several iterations of testing!